### ISSUE TITLE
[Synthese] Url de la source de donnée lorsque l'url est différent de "<monurlsource>/<monid>"

### ISSUE THREAD

**Issue by lpofredc** [PnX-SI/GeoNature#1169]
Les url des sources des données en synthèse sont générés dans le template par ajout d'un `/`+`source_entity_pk_value` à l'a fin de l'url source. Cette solution, bien que commune, ne répond pas toujours à la réalité (ex: dans dbchiro on est sur un format `http://monsite.net/sightings/<monid>/detail` ou dans VisioNature, `https://www.faune-drome.org/index.php?m_id=54&id=<monId>`. Peut-être une piste à creuser côté front avec cette astuce: https://stackoverflow.com/questions/29182244/convert-a-string-to-a-template-string#41015840
Sinon, côté backend avec un fstring?

https://github.com/PnX-SI/GeoNature/blob/f8789dac9ee72088775229d9a839d0bbd419a430/frontend/src/app/syntheseModule/synthese-results/synthese-list/synthese-info-obs/modal-info-obs.component.ts#L75-L77

**Comment by joelclems** [comment]
dans la fonction `backToModule` on pourrait faire:
- si la chaine `<id>` est rpésente dans url_source, on la remplace par la valeur de id_pk_source
http://monsite.net/sightings/<id>/detail
- sinon on ajoute l'id_pk_source à lal fin `url_source``'/' + id_pk_source`

`<id>` parce que c'est simple et donc peu source d'erreur

**Comment by lpofredc** [comment]
Oui, ça me semble une bonne solution, plus facile à mettre en oeuvre.
